server/routes/userEventRouter.ts

```diff
@@ -0,0 +1,20 @@
+import express from "express";
+import {
+	create,
+	listEvents,
+	read,
+	remove,
+	update
+} from "../controllers/userEventController";
+
+const userEventRouter = express.Router();
+
+userEventRouter.post("/create", create);
+userEventRouter.get("/read", read);
+userEventRouter.get("/list", listEvents);
+
+// TODO : convert these two to use path param id if possible
+userEventRouter.put("/update", update);
+userEventRouter.delete("/delete", remove);
```

Comment:
/delete, remove --> name the controller "delete" imo. This is kinda a duplicate of something I already commented.